WordPress/gutenberg
# Editor: Move the panel visibility state from the edit-post to the editor package

Related #52632 

## What?

This PR continues the work on the great unification between post and site editors. The edit-post package holds some state (actions and selectors) that are used to persist the state of the panels (opened, closed, enabled...). In order to use these panels in the site editor as well, we need to unify that state as well by moving it to the editor package. This PR does that.

## Testing instructions

 - Check that you can open and close panels on the sidebar of the post editor
 - Check that their state is persisted upon refresh.
